src/midifile.c

```diff
@@ -456,7 +457,7 @@ static boolean ReadTrack(midi_track_t *track, FILE *stream)
     {
         // Resize the track slightly larger to hold another event:
 
-        new_events = realloc(track->events, 
+        new_events = I_Realloc(track->events, 
```

Comment:
I don't mind either way. It is nice to be consistent across the codebase. In the future, it might be a good idea to add some Travis checks that search for new additions of the banned functions listed in `HACKING.md`, so the more consistent we are in "always just use these", the simpler that will be.